app/src/androidTest/java/com/github/factotum_sdp/factotum/ui/picture/PictureFragmentOnlineTest.kt

```diff
@@ -45,45 +48,61 @@ class PictureFragmentOnlineTest {
     fun setUp() {
         storage = Firebase.storage
         storage.useEmulator("10.0.2.2", 9199)
-        emptyFirebaseStorage(storage)
-
-        // Empty Local Files
         emptyLocalFiles(picturesDir)
 
-        // Launch the fragment
-        onView(ViewMatchers.withId(R.id.drawer_layout))
+        // Open the drawer
+        onView(withId(R.id.drawer_layout))
             .perform(DrawerActions.open())
-        onView(ViewMatchers.withId(R.id.pictureFragment))
+        onView(withId(R.id.roadBookFragment))
             .perform(ViewActions.click())
 
+        // Click on one of the roadbook
+        val destID = DestinationRecords.RECORDS[2].destID
```

Comment:
This code is shared with the PictureFragmentOfflineTest class and could be in a method, so that it's defined only once